### DISCUSSION TITLE
Interesting alternatives

### DISCUSSION THREAD

**Discussion by bryanrcarlson** [CafIncubator/Midden#74] [General]
A listing of (slightly) similar (meta)data catalogs.

- [Dataverse](https://github.com/IQSS/dataverse): "Dataverse is an open source software platform for sharing, finding, citing, and preserving research data"
- [GeoNetwork](https://github.com/geonetwork): "GeoNetwork is a catalog suited to publish information about geospatial (and other) (open and closed) data."
- [Open Science Framework](https://github.com/CenterForOpenScience/osf.io): "OSF is a free, open platform to support your research and enable collaboration."
- [Figshare](https://figshare.com/): "figshare is a repository where users can make all of their research outputs available in a citable, shareable and discoverable manner"
- [OpenMetadata](https://open-metadata.org/): "A Single Place to Discover, Collaborate and get your Data Right. Unlock the value of data assets with an end-to-end metadata management solution that includes data discovery, governance, data quality, observability, and people collaboration."

**Comment by riemino** [comment]
Bryan, to me the most comparable tool to Midden from your list is OSF.  Trying to still understand in my head how both tools could coexist. OSF makes sense to me for project documentation.  In fact, maybe we could come up with a recommendations of using this and Midden instead of using Midden for both dataset and project documentation.

**Comment by bryanrcarlson** [comment]
Interesting Reddit discussion: https://www.reddit.com/r/dataengineering/comments/yxrh9y/metadata_store_which_one_to_choose_openmetadata/
